locales/en/plugin__kubevirt-plugin.json

```diff
@@ -439,5 +439,8 @@
   "MAC Address": "MAC Address",
   "masquerade": "masquerade",
   "VMI": "VMI",
-  "Virtual Machines Instances": "Virtual Machines Instances"
+  "Virtual Machines Instances": "Virtual Machines Instances",
```

Comment:
turns out we are missing some extractions in `main`.
hijacking this PR to fix

Reply:
should be `VirtualMachinesInstances` it's a CRD name.